var/da/da_radiance/da_read_obs_fy3c.inc

```diff
@@ -0,0 +1,801 @@
+subroutine da_read_obs_fy3c (iv, infile_tb)
+   !--------------------------------------------------------
+   !  Purpose: read in FY3C MWHS2 Level-1 data in HDF5 format
+   !           and form innovation structure
+   !
+   !   METHOD: use F90 sequantial data structure to avoid read the file twice
```

Comment:
Change to "   !   METHOD: use F90 sequential data structure to avoid reading the file twice"

Reply:
The sentence has been changed according to your suggestion. Thanks.